src/api/app/views/webui2/webui/search/owner.html.haml

```diff
@@ -4,3 +4,47 @@
   = render partial: 'tabs'
   .card-body
     %h3 Search for people responsible for package:
+    .d-flex.justify-content-center
+      = form_tag(search_owner_path, method: :get, class: 'my-3 w-75') do
+        %input{ name: 'owner', type: 'hidden', value: '1' }
+        %input{ name: 'package', type: 'hidden', value: '0' }
+        %input{ name: 'project', type: 'hidden', value: '0' }
+        .form-group.d-flex
+          = text_field_tag('search_text', params[:search_text], placeholder: 'Search', autofocus: true, required: true, class: 'form-control')
+          %button.btn.btn-primary.ml-1{ type: 'submit', title: 'Search' }
+            %i.fa.fa-search
+        .form-row
+          .d-flex.flex-wrap.flex-column.flex-sm-row
+            .form-group.form-inline
+              .custom-control.custom-switch.ml-1.mr-4
+                = hidden_field_tag('devel', 'off', id: 'nodevel')
+                = check_box_tag('devel', 'on', @owner_devel.nil? || @owner_devel == '1', id: 'devel', class: 'custom-control-input')
+                %label.custom-control-label{ for: 'devel' } Follow Devel Projects
+
+            .form-group.form-inline
+              - limit_options = options_for_select([['Newest', '1'], ['Oldest', '-1'], ['All', '0']], @owner_limit)
+              = select_tag('limit', limit_options, class: 'custom-select border-0 pl-0 w-auto')
+              %label#limit-label{ for: 'limit' }
+                = 'package'.pluralize(@owner_limit.to_i.abs)
+    -# TODO: Ajaxify this section, so whenever changing the filters above, it fetches and refreshes the results
+    - if @results.present?
+      = render partial: 'results_owner', locals: { results: @results }
+      - if User.session
+        %ul.nav
+          %li.nav-item
+            = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#request-bugowner-change-modal' }) do
+              %i.fas.fa-exchange-alt.text-primary
+              Request bugowner change
+        - projects = @results.map { |result| { project: result.project, package: result.package } }
```

Comment:
calling projects to an array of hashes with keys project and package is confusing

Reply:
True... I'll find a better name

Reply:
I went with projects_and_packages. It's longer, but it is what it is...